module/move/wca/tests/inc/commands_aggregator/basic.rs

```diff
@@ -228,6 +228,60 @@ tests_impls!
       "Unexpected validation error type, expected ValidationError::ExecutorConverter."
     );
   }
+
+  // tests bug fix when passing a subject with a colon character
+  // example: passing the path to a directory with a colon in its name
+  fn subject_with_colon() 
+  {
+    let grammar = GrammarConverter::former()
+    .command
+    (
+      wca::Command::former()
+      .hint( "hint" )
+      .long_hint( "long_hint" )
+      .phrase( "command" )
+      .subject( "A path to directory.", wca::Type::Path, true )
+      .form()
+    )
+    .form();
+
+    let executor = ExecutorConverter::former()
+    .routine( "command", Routine::new( | _ | { println!( "hello" ); Ok( () ) } ) )
+    .form();
+
+    let ca = CommandsAggregator::former()
+    .grammar_converter( grammar )
+    .executor_converter( executor )
+    .build();
+
+    let command = r#".command "./path:to_dir" "#;
```

Comment:
The issue is to get the same string. Since `std::env::args` returns `Vec<String>`, and we don't know where to put the quotes

Reply:
Could you also write a test with possible problems? For example, if a property name is `D` and the value is `something`, we might not be able to determine it correctly, if a subject has `:` and is expected as a string value, etc

Reply:
Still relevant. 
We cannot make `.command "./path:to_dir"` from `std::env::args` which returns `[".command", "./path:to_dir"]`. 
What do you think about that?